e2e-tests/admin/tests/login.spec.js

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { test } from '../playwright.config.js';
+import { test } from '../../global-setup.js';
 import { expect } from '@playwright/test';
 
 import { LoginPage } from '../pages/login.js';
```

Comment:
this is a nitpick- should it say `resources` instead of pages?

Reply:
No, this is a playwright [convention](https://playwright.dev/docs/pom) for organizing actions